packages/fxa-auth-server/lib/senders/emails/layouts/fxa/index.ts

```diff
@@ -40,9 +40,9 @@ export const render = (body: string) => {
       ${body}
       <mj-section>
         <mj-column>
-          <mj-text css-class="secondary-text">Mozilla. 2 Harrison St, #175, San Francisco, CA 94105</mj-text>
-          <mj-text css-class="secondary-text"><a class="link-blue" data-l10n-id="fxa-privacy-url" href="<%- privacyUrl %>">Mozilla Privacy Policy</a></mj-text>
-          <mj-text css-class="secondary-text"><a class="link-blue" data-l10n-id="fxa-service-url" href="https://www.mozilla.org/about/legal/terms/services/">Firefox Cloud Terms of Service</a></mj-text>
```

Comment:
Even I agree these classnames do look a little weird so it makes sense to assign them better names. Assuming there'd be more sections in the bodytext in future template conversions, I gave them names according to what underlying font size is being used and since it is used for `<mj-text>` elements, it made sense to start the classname with `text`